message.d.ts

```diff
@@ -400,6 +400,11 @@ export namespace MessageEntity {
     /** For “pre” only, the programming language of the entity text */
     language?: string;
   }
+  export interface CustomEmojiMessageEntity extends AbstractMessageEntity {
+    type: "custom_emoji";
+    /** For “custom_emoji” only, unique identifier of the custom emoji. Use getCustomEmojiStickers to get full information about the sticker */
+    custom_emoji: string;
+  }
```

Comment:
This is missing in the union type.

Also, as the number of entities grows, I think I'd prefer to keep them sorted in the same order as in the union type, which is alphabetically.